src/main/java/de/moonflower/jfritz/box/fritzbox/FritzBox.java

```diff
@@ -1325,9 +1325,11 @@ public void doCall(PhoneNumberOld number, Port port) {
 					log.debug("doCall: Firmware is greater/or equal than 06.01");
 					
 					log.debug("doCall: Setting dialing port to " + port.getDialPort());
-				   	generateDoCallPostDataDialPortLua(postdata, port.getDialPort());
-				   	fbc.postToPageAndGetAsString(URL_DIAL_FONBOOK_LUA, postdata);
-					
+					if (!port.getDialPort().equals(getQueryDialPort())) { // 18.11.2017 Nur wenn Dialport nicht 50=50 ist
```

Comment:
Hallo Erwin, 
was ist der Sinn dieser if-Bedingung? Dass der Port nur gesetzt wird, wenn er sich geändert hat?

Grüße,
Robert

Reply:
Nein es Führt nicht zu Problemen aber das Wählen dauert länger (3-5 Sec) wenn man jedes
mal den Port setzt was für mich keinen sinn macht, und wenn die Sicherheits Funktion in der Box 
Aktiv ist dann kann man eh kein Post senden.

https://www.ip-phone-forum.de/threads/jfritz-einschr%C3%A4nkung-nach-fritzbox-update.297242/page-2#post-2249962

Gruß Erwin